sdk/keyvault/keyvault/inc/az_keyvault.h

```diff
@@ -5,43 +5,103 @@
 #define AZ_KEYVAULT_H
 
 #include <az_contract.h>
+#include <az_credential.h>
+#include <az_http_pipeline.h>
+#include <az_http_request_builder.h>
 #include <az_result.h>
 #include <az_span.h>
+#include <az_str.h>
 
 #include <stdlib.h>
 
 #include <_az_cfg_prefix.h>
 
-typedef struct {
-  az_span uri;
-  // TODO: Azure Credentials
-} az_keyvault_keys_client;
+extern az_span const AZ_KEY_VAULT_NONE_TYPE;
+extern az_span const AZ_KEY_VAULT_KEY_TYPE;
+extern az_span const AZ_KEY_VAULT_SECRET_TYPE;
+extern az_span const AZ_KEY_VAULT_CERTIFICATE_TYPE;
+extern az_span const AZ_KEY_VAULT_URL_PATH_SEPARATOR;
```

Comment:
If we really need/want this, then I think it can be global: All URLs use the same path separator '/'.

Reply:
Agree. I would remove this const and use `AZ_STR("/")` if needed.